Report.md

```diff
@@ -123,7 +123,21 @@
 ![Image of Travel Company Empathy Map](./diagrams/ADinboundcustomer.png)
 
 ## 7. Discuss the competitive advantages might be gained in developing the new information system. Identify and discuss the possible adverse effects for this Business if its information system project fails
+<ins>*Competitive advantages*</ins>
 
+Developing this new information system, if successful, will lead to a plethora of competitive advantages against other major travel companies looking to increase their customer loyalty or retention. Currently, the major travel company involved is in a great position in the market but has some areas that it is lacking in. For example, the CMC (Call Management Centre) has reduced operating efficiency due to call flow control and call routing issues. This leads to some customers having long wait times and drives up the inbound call cost of the CMC. Through the information system, these areas could be improved and provide many advantages.
+
+If the call flow control and routing issues are removed or even lessened through the development of the new information system, then customers will have shorter wait times as well as more effective matches with RMs (Relationship Managers). This increase in user experience will improve customer retention, as they see changes being made in the system for their benefit. Not only this, but by directly making changes to areas they are displeased with (and implementing an easier method of feedback elicitation for future, subpar areas), the travel company stands to gain a large uptake in customer loyalty - due to their voices being heard and taken into consideration. This will also inadvertently put greater risk on competitors to keep up, which is an extraordinary competitive advantage.
+
+From a more business-centric point of view, not only with the customers be happier with the system, but the travel company will also save costs in handling the soon-to-be larger customer base. The information system's ability to reduce per-call handling time will reduce the cost of inbound calls to the CMC. This, in combination with the increased income that comes with a greater amount of customers, will even further extend the profit of the travel company.
+
+<ins>*Adverse effects*</ins>
+
+If the information system is ineffective, lacklustre or outright useless, there are some considerable adverse effects to first take into account. Ranging from the loss of customers, as opposed to sharp gain, to increased operating costs for no benefit, the adverse effects possible should be weighed into consideration alongside the competitive advantages to most accurately see the possible result of the information system.
```

Comment:
_fundamentally_ or _inexplicably_ useless perhaps. btw what is `<ins>`

Reply:
<ins> just underlines it
Yep will change

Reply:
I didn't expect it to underline what I wrote ahaha